data_structures/linked_heap.py

```diff
@@ -116,7 +117,22 @@ def merge(new_node: BinaryNode[int, T]):
         res.__root = tree
         return res
 
+    @staticmethod
```

Comment:
If you decide to keep this method in the linked structure, why is it marked as static?